lib/block.rb

```diff
@@ -2,65 +2,23 @@
 
 require_relative './helpers.rb'
 require 'openssl'
-require 'pg'
 
-# Understands creating and retrieving new blocks
+# block class
```

Comment:
Can you write this comment in the 'Understands..' form to fit convention? Just a description of what the class does (sorry, I realise this is minor 😛)